Jenkinsfile-Release

```diff
@@ -36,33 +36,31 @@ dockerizedBuildPipeline(
   deploy: {
     final branch = gitBranch(env)
 
-    sh """
-      git config user.email "sonatype-ci@sonatype.com"
-      git config user.name "Sonatype CI"
-
-      git checkout ${branch}
-      git add docs
-      git commit -m 'Release Update'
-    """
+    runSafely 'git config user.email "sonatype-ci@sonatype.com"'
+    runSafely 'git config user.name "Sonatype CI"'
+    runSafely "git checkout ${branch}"
+    runSafely 'git add docs'
+    runSafely 'git commit -m "Release Update"'    
```

Comment:
i know i've been unclear at times about the different ways to execute external processes. why is runSafely better than sh?

Reply:
The reason I changed it was because `runSafely` will check the OS and run the command using the proper shell, but I just realized this does not make any sense for `helm-charts`, as we are running on a docker environment 😅 

With this in mind do you think it is worth reverting the change to use `sh` again? Maybe that will be cleaner?

Reply:
I would keep `runSafely`. There's no harm in doing that and it looks "safer" :smiley: